src/main/java/frc/robot/subsystems/StateMachine.java

```diff
@@ -209,8 +210,13 @@ else if (m_state == State.POOP_SCORE)
   }
 
   public Command dragonScoreSequence() {
-    return m_dragon.score()
-        .beforeStarting(() -> m_state = State.DRAGON_SCORE);
+    if (m_elevator.getSetpoint() == ElevatorSetpoint.L4) {
+      return m_dragon.scoreRetract()
+          .beforeStarting(() -> m_state = State.DRAGON_SCORE);
```

Comment:
this should score then retract once the beam gets restored i.e. the coral is off the dragon